### PR TITLE
WIP: dependencies: bump kube-openapi

### PR DESCRIPTION
**Pull request by pohly**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Quite a few [changes have accumulated in kube-openapi](https://github.com/kubernetes/kube-openapi/compare/70dd3763d340...76de80e0abd9), of which at least one changes the output (see second commit).

#### Special notes for your reviewer:

Perhaps https://github.com/kubernetes/kube-openapi/commit/5cb5474018333877c24c0fa1a73963d4c85dd075 caused the changes in the output?

cc @Jefftree @devholic @alexzielenski 

#### Does this PR introduce a user-facing change?
```release-note
OpenAPI: quoting of backslashes changed
```

### CODE REVIEW

**Comment by pohly** [comment]
Besides the change in the output, this update also adds [unwanted dependencies](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/126917/pull-kubernetes-dependencies/1828004773400416256).

File: vendor/github.com/go-openapi/swag/string_bytes.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package swag
+
+import "unsafe"
```
Comment (liggitt) [inline vendor/github.com/go-openapi/swag/string_bytes.go:3]: do we know where and how this is used? just want to make sure nothing is sharing byte data in ways that break our use of shared openapi objects

Comment (liggitt) [reply vendor/github.com/go-openapi/swag/string_bytes.go:3]: (sorry... wrong PR, I meant to ask this on https://github.com/kubernetes/kubernetes/pull/126787)

**Comment by pohly** [comment]
https://github.com/kubernetes/gengo/commit/239d3d48564a0b8e2bf8a60dc7734d33b5a89575 changed the API so that `staging/src/k8s.io/code-generator` must be updated. On it...

**Comment by liggitt** [comment]
> [kubernetes/gengo@239d3d4](https://github.com/kubernetes/gengo/commit/239d3d48564a0b8e2bf8a60dc7734d33b5a89575) changed the API so that `staging/src/k8s.io/code-generator` must be updated. On it...

@Jefftree is already working through that in https://github.com/kubernetes/kubernetes/pull/126787, could sync with him to avoid duplicate work

**Comment by pohly** [comment]
/close

In favor of https://github.com/kubernetes/kubernetes/pull/126787

**Pull request closed without merging**